commit c8f101e4cca0092645070480c2837c446894f305
Author: hasufell <hasufell@gentoo.org>
Date:   Thu Jun 20 20:38:23 2013 +0200

    remove stupid uninstall snippet
    
    whoever coded this... it is dangerous
    uninstall methods have NO PLACE in install rules

diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index cd8106a..3a54dd0 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -268,12 +268,6 @@ endif()
 # Install Targets (Generic, All Platforms)
 
 
-# important to make a clean  install each time, else old scripts get loaded.
-install(
-	CODE
-	"file(REMOVE_RECURSE ${TARGETDIR_VER})"
-)
-
 if(WITH_PYTHON)
 	# install(CODE "message(\"copying blender scripts...\")")
 	
